python/GafferUI/NodeGraph.py

```diff
@@ -285,9 +294,11 @@ def __keyPress( self, widget, event ) :
 		elif event.key == "Down" :
 			selection = self.scriptNode().selection()
 			if selection.size() :
-				needsModifiers = not isinstance( selection[0], ( Gaffer.Reference, Gaffer.Box ) )
-				haveModifiers = bool( event.modifiers & ( event.modifiers.Shift | event.modifiers.Control ) )
-				if needsModifiers == haveModifiers :
+				needsModifiers = not Gaffer.Metadata.value( selection[0], "nodeGraph:childrenViewable" )
+				if (
+					( needsModifiers and event.modifiers == event.modifiers.Shift | event.modifiers.Control ) or
```

Comment:
The version we just pushed out already has the change whereby folks will need to unlearn that, because in it you can only get into regular Boxes _without_ modifiers.